### PR TITLE
Release/1.1.0

### PR DESCRIPTION
**Pull request by Jrobles70**
### Updated
 - Compatibility with Python2.7

### Fixed
 - Lazy load list slice
 - Patch tests that were failing

### Added
 - Auth class
 - Scope property for Client and Affiliate
 - Parent Scope property for Affiliate
 - User class that supports retrieve, list, and update.

### CODE REVIEW

**Comment by Jrobles70** [comment]
We should add the buddy pipelines as a required passing before merging. We used to have them but the status checks arent available in the repo settings anymore.

File: ticketguardian/order/order.py [hunk 1]
```diff
@@ -14,11 +15,15 @@
...
         ListResourceMixin,
         RetrieveResourceMixin,
+        PatchResourceMixin,
         PostResourceMixin,
         PutResourceMixin, ):
 
     resource = "orders"
 
+    def __init__(self):
+        super(Order, self).__init__()
```
Comment (austinphilp) [inline ticketguardian/order/order.py]: Why is this necessary? Wouldn't the super method also be called if you just removed the method entirely?

Comment (austinphilp) [reply ticketguardian/order/order.py]: There are a few instances of this all over, can I assume this is a python2 compatibility thing?
